docs/static/spec/openapi/logstash-api.yaml

```
@@ -32,52 +32,195 @@ security:
+      parameters:
+        - name: pretty
+          in: query
+          schema:
+            type: boolean
+          description: >
+            If you append `?pretty=true` to the request, the JSON returned will be pretty formatted. Use it for debugging only!
+      responses:
+        '200':
+          description: Indicates a successful call
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/Common'
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - type: object
+                    properties:
+                      os:
+                        type: object
+                        properties:
+                          name:
+                            type: string
+                          arch:
+                            type: string
+                          version:
+                            type: string
+                          available_processors:
+                            type: integer
+
+  /_node/pipelines:
+    get:
+      summary: Get pipeline-specific information
+      description: |
+        Gets pipeline-specific information and settings for each pipeline.
+      operationId: nodeInfoPipelines
+      tags:
+        - node info
+      parameters:
+        - name: pretty
+          in: query
+          schema:
+            type: boolean
+          description: >
+            If you append `?pretty=true` to the request, the JSON returned will be pretty formatted. Use it for debugging only!
+      responses:
+        '200':
+          description: Indicates a successful call
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/Common'
+                  - $ref: '#/components/schemas/PipelineSettings'
+                  - type: object
+                    properties:
+                      pipelines:
+                        type: object
+                        additionalProperties:
+                          $ref: '#/components/schemas/NodeInfoPipeline'
+              examples:
+                nodeInfoPipelineExample1:
+                  # summary:
+                  value:
+                    pipelines:
+                      test:
+                        workers: 1
+                        batch_size: 1
+                        batch_delay: 5
+                        config_reload_automatic: false
+                        config_reload_interval: 3
+                      test2:
+                        workers: 8
+                        batch_size: 125
+                        batch_delay: 5
+                        config_reload_automatic: false
+                        config_reload_interval: 3
 
-        - `pipelines`
-        - `os`
-        - `jvm`
-      operationId: nodeInfo
+  /_node/pipelines/{pipeline_name}:
+    get:
+      summary: Get a specific pipeline information
```

There are some other variations we've used too like "Get details about a pipeline"
```suggestion
      summary: Get information for a pipeline
```